lib/resty/websocket/client.lua

```diff
@@ -236,7 +232,11 @@ function _M.connect(self, uri, opts)
         return nil, "bad HTTP response status line: " .. header
     end
 
-    return 1
+    if not keep_response then
+      header = nil
+    end
+
+    return 1, nil, header
```

Comment:
I see - in light of this my comment on the previous commit is outdated. This perhaps could be accepted maybe assuming some renaming.